src/components/molecules/NotificationMarkdown/TelemetryButtons.tsx

```diff
@@ -28,8 +26,11 @@ export const TelemetryButtons = ({notificationId}: {notificationId?: string}) =>
   const handleNotOk = () => {
     disableTelemetryNotification();
 
+    /* 
+    Will not open any where since SettingsPanel is available after opening a project
```

Comment:
@devcatalin @topliceanurazvan @manoukyanmari @yasarkubeshop 

Please have a look here

Reply:
Is there a way to check if a project is already opened and only then set the left menu to settings?